profiles/source/Source_4K_Display.yaml

```diff
@@ -1,6 +1,8 @@
 dsDisplay:
   Type: source
   Name: xione
+  Number_of_ports: 0
+  Video_Ports: []
```

Comment:
why do we need Aspect Ratio here? Interface document already defines the default value returned if device in not connected.

Reply:
done the changes